spec/support/app_helpers.rb

```diff
@@ -1,6 +1,9 @@
 module AppHelpers
   def view_manual_change_notes
-    click_on "see all updates"
+    within('header test-govuk-component[data-template="govuk_component-metadata"]') do
```

Comment:
Slimmer provides some test helpers for working with components:
https://github.com/alphagov/slimmer/blob/868095a99fdf559cdad80b9102d882322214bc60/lib/slimmer/test_helpers/shared_templates.rb#L12-L15

Example usage:
https://github.com/alphagov/government-frontend/blob/master/test/test_helper.rb#L33
https://github.com/alphagov/government-frontend/blob/master/test/test_helper.rb#L57-L61